### ISSUE TITLE
pylint 2.14 does not recognise 'no-self-use' option value

### ISSUE THREAD

**Issue by sebastian-tschen** [PyCQA/pylint#6792]
### Bug description

a comment like:
```
    # pylint: disable=no-self-use
```
to disable 'no-self-use' warnings for a method, will lead to an error in 2.14, but not in 2.13.9
```
E0012: Bad option value for disable. Don't recognize message no-self-use. (bad-option-value)
```

### Configuration

```ini
simple example that reproduces the error:

class SomeClass:

    # pylint: disable=no-self-use
    def print_stuff(self, hello):
        print(hello)

def main():
    someinstance = SomeClass()
    someinstance.print_stuff("hello world")


if __name__ == '__main__':
    main()
```
```


### Command used

```shell
pylint fail.py
```

### Pylint output

```shell
************* Module fail
fail.py:3:0: E0012: Bad option value for disable. Don't recognize message no-self-use. (bad-option-value)
fail.py:1:0: C0114: Missing module docstring (missing-module-docstring)
fail.py:1:0: C0115: Missing class docstring (missing-class-docstring)
fail.py:4:4: C0116: Missing function or method docstring (missing-function-docstring)
fail.py:1:0: R0903: Too few public methods (1/2) (too-few-public-methods)
fail.py:7:0: C0116: Missing function or method docstring (missing-function-docstring)

-----------------------------------
Your code has been rated at 0.00/10
```

### Expected behavior

don't recognize an error.

### Pylint version

```shell
pylint 2.14.0
astroid 2.11.5
Python 3.9.0 (default, Jun 30 2021, 09:39:40) 
[GCC 8.3.0]
```

### OS / Environment

debian
